src/cljs/rems/administration/create_form.cljs

```diff
@@ -8,20 +8,39 @@
             [rems.atoms :refer [document-title]]
             [rems.collapsible :as collapsible]
             [rems.fields :as fields]
+            [rems.spinner :as spinner]
             [rems.status-modal :as status-modal]
             [rems.text :refer [text text-format]]
-            [rems.util :refer [dispatch! post! normalize-option-key parse-int remove-empty-keys]]))
+            [rems.util :refer [dispatch! fetch post! normalize-option-key parse-int remove-empty-keys]]))
 
 (rf/reg-event-fx
  ::enter-page
- (fn [{:keys [db]}]
-   {:db (assoc db ::form {:fields []})}))
+ (fn [{:keys [db]} [_ form-id]]
+   {:db (assoc db
+               ::form {:fields []}
+               ::loading? (not (nil? form-id)))
```

Comment:
I'd prefer setting `::loading?` inside `::fetch-form` to improve cohesion. See how rems/catalogue.cljs does `::fetch-catalogue`.

Reply:
Done.